dds/DCPS/transport/shmem/ShmemTransport.cpp

```diff
@@ -67,12 +67,12 @@ ShmemTransport::connect_datalink(const RemoteTransport& remote,
   ShmemDataLinkMap::iterator iter = links_.find(key.second);
   if (iter != links_.end()) {
     ShmemDataLink_rch link = iter->second;
-    VDBG_LVL((LM_DEBUG, ACE_TEXT("(%P|%t) ShmemTransport::connect_datalink ")
-              ACE_TEXT("link found.\n")), 2);
+    ACE_DEBUG((LM_DEBUG, ACE_TEXT("(%P|%t) ShmemTransport::connect_datalink ")
```

Comment:
Do we want this to be an unconditional log message?  (same on line 74)

Reply:
Whoops, missed changing this one back from debugging.